doc/nrf/device_guides/working_with_nrf/nrf70/developing/fw_patches_ext_flash.rst

```diff
@@ -0,0 +1,140 @@
+
+Overview
+********
+
+By default the nRF70 Series firmware patches are built as part of the nRF Wi-Fi host driver code, residing in on-chip memory.
+The firmware patches include code that is executed on the nRF70 Series device.
+The size of the firmware patches may be considerably large, which limits the amount of on-chip code memory available for the user application.
+In order to increase the amount of on-chip memory available for user applications, the nRF70 Series Wi-Fi driver supports the option of using external memory, if that is available.
+
+Prerequisites
+=============
+
+Before using this feature, make sure that the following prerequisites are completed:
+
+ * The external memory must be available and configured in the devicetree.
+ * The external memory must be large enough to store the firmware patches, taking into consideration also the patch upgrade requirement, for example, Device Firmware Update (DFU).
+   The maximum size of all the firmware patches combined is 128 KB.
+
+.. note::
+    Currently, in Nordic Semiconductor SoCs, access to external memory and the Execute in Place (XIP) feature are available through the QSPI interface only.
+
+Supported platforms
+===================
+
+The following platforms are supported:
+
+* nRF5340 SoC or nRF5340 DK with nRF7002 EK as a shield
+* nRF52840 SoC or nRF52840 DK with nRF7002 EK as a shield
+
+.. note::
+    Due to limitations mentioned above, the nRF7002 DK is not supported as it needs access to external memory through the SPI interface.
+
+Available options
+*****************
+
+There are two ways to use the firmware patches in the external memory feature:
+
+* XIP supported external memory
+* Non-XIP supported external memory
+
+
+XIP supported external memory
+=============================
+
+If the application supports XIP from external memory, then the firmware patches can be loaded as part of the nRF Wi-Fi driver code and then relocated to the external memory and executed from there.
+To enable this feature, set the :kconfig:option:`CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP` kconfig option to ``y``.
+Once the build is complete, the feature can be verified by checking the memory regions summary in the build output.
+A new memory region called ``EXTFLASH`` is added to the memory regions summary, and the firmware patches are placed in this region.
+The size of the ``FLASH`` used is reduced by the size of the firmware patches.
+
+A sample memory regions summary is shown below:
+
+.. code-block:: console
+
+  Memory region         Used Size  Region Size  %age Used
+        EXTFLASH:       64544 B         8 MB      0.77%
+           FLASH:      618752 B         1 MB     59.01%
+             RAM:      170636 B       448 KB     37.20%
+        IDT_LIST:          0 GB         2 KB      0.00%
+
+.. note::
+    Depending on XIP implementation, this can cause a performance impact in both time and power.
```

Comment:
While this is correct, I would prefer this being a separate section, where we can also discuss best practices. For example, we can mention the overheads
- firmware upload time
- XIP flash energy overhead
and discuss what the recommended practice is. Somewhere we need to tell the users that they need to enable and disable the XIP after the patch upload is complete...

Reply:
We don't have those profiling numbers in hand, we need to schedule a task to collect these (and figure out how to). IMHO, we should start with a generic note and once we have specific benchmarks we can move this to a separate section.

Reply:
I meant, the note could be moved out of this section, and put into a separate "Notes" or "Best practices" section where we could - in the future - add more context. I think the note here is not helping much the reader, unless we give them more context...